packages/plugin-ext/src/hosted/node/plugin-reader.ts

```diff
@@ -50,8 +50,13 @@ export class HostedPluginReader implements BackendApplicationContribution {
 
             const localPath = this.pluginsIdsFiles.get(pluginId);
             if (localPath) {
-                res.sendFile(filePath, { root: localPath }, (error: Error) => {
-                    res.status(404).send(`No such file for plugin with id '${escape_html(pluginId)}'.`);
+                res.sendFile(filePath, { root: localPath }, e => {
+                    if (e) {
+                        console.error(`Could not find '${filePath}' file for '${pluginId}'`, e);
```

Comment:
Is this the only thing that can go wrong when sending the file? 404 is for file not found, but can other things go wrong that should be mapped to different error codes?

Reply:
We could map `ENOENT` to 404, and the rest to `500` (internal server error). I will do it.